pkg/api/networkservice/constants.go

```diff
@@ -23,4 +23,9 @@ const (
 	PodNameKey = "podName"
 	// NamespaceKey - namespace a container is running in
 	NamespaceKey = "namespace"
+
+	// DNSConfigShouldNotBeNil -
+	DNSConfigShouldNotBeNil = "dnsConfig should not be nil"
+	// DNSServerIpsShouldHaveRecords -
+	DNSServerIpsShouldHaveRecords = "dnsConfig should have records"
```

Comment:
Why we need these?

Reply:
@denis-tingajkin could you comment? @nickolaev  also I think we should not introduce fixes into this move pieces PR, but do all changes in separate ones. Thanks.

Reply:
Yeah... I have a lot of my own nits I want to fix too, but intentionally kept this a straight move